src/core/search/indices.cc

```diff
@@ -8,12 +8,14 @@
 #include <absl/strings/ascii.h>
 #include <absl/strings/numbers.h>
 #include <absl/strings/str_split.h>
-#include <unicode/brkiter.h>
-#include <unicode/unistr.h>
+
+#define UNI_ALGO_DISABLE_NFKC_NFKD
```

Comment:
maybe add a comment explaining why you disabled this.

Reply:
Because its not needed, just so to reduce header parsing times 🤷🏻